lib/jxl/modular/transform/enc_transform.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#include "lib/jxl/fields.h"
+#include "lib/jxl/modular/modular_image.h"
+#include "lib/jxl/modular/transform/transform.h"
+
+namespace jxl {
+
+Status FwdRCT(Image &input, size_t begin_c, size_t rct_type);
```

Comment:
this normally goes in a enc_rct.h header. Elsewhere we have the enc_foo.h header include the dec_foo.h header or just foo.h header if you want.

Reply:
Yes, that would be the logical thing to do, but I felt it was a bit overkill to add 3 more .h files each with just 1 function prototype in them (and that being unlikely to change). I'll do it if you insist though :)

Reply:
Could also do enc_transforms.h and dec_transforms.h and put everything in those :)

Reply:
I prefer keeping the 3 transforms in separate files, it's a bit much code to put it all together, easier to edit this way.

Reply:
It helps with understanding where are things if you have the .h file, specially if you are unfamiliar with the project. I'd find it surprising that enc_transform.cc depends on enc_rct.cc but there's no include of enc_rct.h; when dealing with stripping out dependencies I move around code and you use these explicit dependencies to understand what depends on what and what can be removed or moved to another library.

Reply:
OK, makes sense.

Reply:
Better now?

Reply:
thanks